meshapp/src/profile/profile-page.tsx

```diff
@@ -290,7 +307,34 @@ const ProfileEducation = (props: any) => {
         >
           Education
         </Typography>
-        <TestComponent />
+        <ProfileGroupAccordion
+          groupAccordArgs={props.education.map((currentEd) => {
+            return {
+              headerOne: currentEd.degree,
+              headerTwo: currentEd.school,
+              descText: currentEd.description,
+            };
+          })}
+          headerOnePlaceholder="Level of Education"
+          headerTwoPlaceholder="School"
+          headerOneOptions={[
+            "High School Diploma",
+            "Associates Degree",
+            "Bachelors Degree",
+          ]}
+          headerTwoOptions={["Cal Poly Pomona", "Mt. San Antonio College"]}
+          descPlaceholder="Enter a description here"
+          headerOneErrValidations={[
+            isEmpty("Level of Education"),
+            whiteSpace("Level of Education"),
+          ]}
+          headerTwoErrValidations={[isEmpty("School"), whiteSpace("School")]}
+          descErrValidations={[
+            isEmpty("Description"),
```

Comment:
Do we really need a check for if the description is empty? I feel that this should be some optional thing to add the description, not necessarily be forced on the user.

Reply:
removed "isEmpty("description")" so now it should not check for empty description;  description can now be empty (commit #16)